modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/graphql/data/processor/LiferayMethodDataFetchingProcessor.java

```diff
@@ -129,16 +141,16 @@ public Object process(
 
 			Object argument = arguments.get(parameterName);
 
-			if (argument == null) {
+			if (parameterName.equals("page")) {
```

Comment:
I think this if clause should keep the same format as the following ones (comparing the parameterName)

Reply:
In this case, we are only comparing the `paramterName` because `argument` property is not necessary. So I moved it separately in different if clauses like this:

```java
if (parameterName.equals("page")) {
	argument = pagination.getPage();
}

if (parameterName.equals("pageSize")) {
	argument = pagination.getPageSize();
}

```
but the SF is complains that they have to be combined the if-else with if clause of `(argument==null)`

Reply:
what do you think @ccorreagg ?

Reply:
I have tried it and the SF is not complaining in my case.. Could you include the code and the SF complaint?

Reply:
```

			if (argument == null) {
				if (parameter.isAnnotationPresent(NotNull.class)) {
					throw new ValidationException(parameterName + " is null");
				}
			}

			if (parameterName.equals("page")) {
				argument = pagination.getPage();
			}
			if  (parameterName.equals("pageSize")) {
				argument = pagination.getPageSize();
			}
```

Reply:
something like this, because the `argument` property is not going to take into account because you are calculating the Pagination object before

Reply:
I guess SF is complaining because of the first two ifs, they should be done in the same line

Reply:
you are right, I was thinking about the three ones together :smile: , thanks for the tip!